src/ayab/encoders.cpp

```diff
@@ -51,15 +51,72 @@ void Encoders::encA_interrupt() {
  * \brief Read hall sensor on left and right.
  * \param pSensor Which sensor to read (left or right).
  */
-uint16_t Encoders::getHallValue(Direction_t pSensor) {
+HallState_t Encoders::getHallValue(Direction_t pSensor) {
```

Comment:
As a HallState is return from now on, we _could_ rename the method to getHallState instead of getHallValue.

Reply:
Sure, I think that makes sense.